x-element.js

```diff
@@ -1107,7 +1107,7 @@ class Template {
               html = html.slice(0, -3 - name.length) + `x-element-property-$${iii}="${name}`;
               state.index = 1; // Accounts for an expected quote character next.
             } else {
-              throw new Error(`Found invalid template near "${string.slice(state.index)}".`);
+              throw new Error(`Found invalid template string "${string}" at "${string.slice(state.index)}".`);
```

Comment:
Snuck this in as well — there are cases where having the additional, full string is valuable. I stopped short of including the entire template string as that can get rather long.